LayoutTests/fast/dom/NodeIterator/NodeIterator-dont-overcollect.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="../../../resources/js-test-pre.js"></script>
+<script src="../../../resources/gc.js"></script>
+<script>
+    description("This test asserts that NodeItrator's callback is kept alive while NodeIterator is alive even if all JS references go away.")
```

Comment:
There is a typo here in one case: `NodeItrator`.

Reply:
Fixed.